Checked off Michael Kochevar

mkhaas6 opened the pull request:
Checked off Michael Kochevar

mkhaas6 committed:
Checked off Michael Kochevar

Checked off Michael Kochevar

henriquedc1 reviewed (commented):
Hi @mkhaas6! Good job checking off your name. However, following your name, we need your username (see Joe's example as a reference). Can you adjust it and save another commit, please? Thanks!

mkhaas6 committed:
Update readme.md

@henriquedc1.  Hi Henrique - I'm updating the request to include my username.  Please let me know if other changes are needed.  Thank you!
